src/isElement.js

```diff
@@ -5,13 +5,17 @@ export function isElement( el )
 {
   let isElem;
 
-  if ( typeof HTMLElement === 'object' )
+  if ( el && el.tagName !== 'HTML' )
```

Comment:
That will fail in certain browsers. e.g. in Firefox the tag name of the `<html></html>` tag is `html`, not `HTML`